shotgun_api3/shotgun.py

```diff
@@ -321,6 +327,10 @@ def __init__(self,
         the server. If password is provided, then login must be as well and
         neither script_name nor api_key can be provided.
         
+        :param auth_token: The one time token required to authenticate to
+        a server with 2FA turned on. If auth_token is provided, then login and password
+        must be as well and neither script_name nor api_key can be provided.
```

Comment:
There isn't a check for script_name + api_key + auth_token by my read.

A call that specifies all three looks like it wouldn't raise an error but would just ignore the auth_token.  Either that should be fixed or this docstring should be updated.